modules/bridge/src/main/ts/ephox/bridge/core/DataProcessors.ts

```diff
@@ -17,7 +17,12 @@ import { dialogToggleMenuItemDataProcessor } from '../components/dialog/ToggleMe
 import { urlInputDataProcessor } from '../components/dialog/UrlInput';
 import { getAllObjects } from './ObjUtils';
 
-const isNamedItem = (obj) => Type.isString(obj.type) && Type.isString(obj.name);
+interface NamedItem {
+  readonly name: string;
+  readonly type: string;
+}
+
+const isNamedItem = (obj: any): obj is NamedItem => Type.isString(obj.type) && Type.isString(obj.name);
```

Comment:
maybe `obj: unknown` instead? Also, not in scope but should we be checking that accessing properties on `obj` won't throw exceptions? Or can we indicate that with the types by saying something like `obj: Object` or something?

Reply:
`any` is actually correct here, as we want to allow any value to be passed in. Additionally `unknown` would throw compiler warnings about accessing unknown properties. Ideally, it should be `Object` or similar but that would mean it would also fail to compile with unknown properties.

Either way, this is largely out of scope as this is currently how we do this everywhere else, so if we want to improve on that we should do it in another PR.